src/components/access_group/AccessGroupMembers.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <article class="members-container">
+    <header class="members-container__top-bar">
+      <SearchForm
+        class="top-bar__search"
+        id="access-group-members-search"
+        v-on:clear-query="clearSearchHandler"
+        :searchFormHandler="searchFormHandler"
+        searchFormLabel="Search Members"
+      ></SearchForm>
+      <Tooltip
+        class="members-container__tooltip"
+        buttonText="Member search info"
+        alternateButtonText="Close member search info"
+        >You can search by name, username and email</Tooltip
+      >
+    </header>
+    <nav class="members-container__tabs">
+      <ul class="tabs-container">
+        <li
+          :class="{ 'tabs-container__item': true, active: isTabActive(tab) }"
+          v-for="(tab, idx) in tabList"
+          :key="idx"
+          @click="handleTabClick(tab)"
+        >
+          {{ tab.label }}
+        </li>
+      </ul>
+    </nav>
+    <ul class="members-container__list">
```

Comment:
We can add a sort by role (i.e. curators are returned in the first pages) to the API. Then, and also in general I think this should just be a flex container and not a list of lists. Right now it breaks in mobile (the lists show underneath each other an curators are separated).

Reply:
I agree that this should be a flex box, but @HerminaC had requested that the whole row not drop when one expands. That was the way I could think of to make this happen. But I can see how this messes things up in mobile. I'll rethink.